src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -1,16 +1,18 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <UsingTask TaskName="Xamarin.Android.Tools.BootstrapTasks.GenerateProfile" AssemblyFile="..\..\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll" />
-  <Target Name="_Foo" BeforeTargets="CoreCompile">
+  <ItemGroup>
+    <SharedRuntimeBuildPath Include="..\..\bin\$(Configuration)\lib\xbuild-frameworks\MonoAndroid\" />
```

Comment:
`@(SharedRuntimeBuildPath)` should have a `_` prefix, as it's not an overridable value.

Additionally, it looks like it should be a property, given that you're not using any `**\*` expansion. Doing so would *also* shorten the `@(_SharedRuntimeAssemblies)` group:

    <_SharedRuntimeAssemblies Include="$(_SharedRuntimeBuildPath)\v1.0\*.dll;$(_SharedRuntimeBuildPath)\$(AndroidFrameworkVersion)\*.dll" />

Additionally, this shouldn't hardcode `v6.0`, it should instead use `$(AndroidFrameworkVersion)`, as done above, because if the developer overrides the `$(AndroidFrameworkVersion)` value in `Configuration.Override.props`, the `v6.0` directory reference won't exist.